.github/issue_template.md

```diff
@@ -0,0 +1,8 @@
+If fixing bug, please document how to reproduce it.
```

Comment:
If fixing a bug, please document how to reproduce it.

Please always test the latest master.

Finding the commit which introduced the problem helps (bisecting).

Document the kernel and distribution that were used.




If adding a new feature, explain why it's useful.





Tests should ideally use network namespaces to not interfere with the rest of the system.